graphql/schema.graphql

```diff
@@ -15,14 +15,25 @@ input DateRange {
 type User {
     name_first: String!
     name_last: String!
-    id: ID
+    id: ID!
     atcRating: Rating!
 }
 
 type Rating {
     code: String!
 }
 
+interface Node {
+    id: ID!
+}
+
+type Query {
+    node(
```

Comment:
What is this?

Reply:
https://atheros.ai/blog/graphql-interfaces-and-unions-how-to-design-graphql-schema have a gander at this